wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -6,39 +6,54 @@ ul.sequence {
 li.sequence-member {
     @include clearfix;
     position: relative;
-    border: 1px solid transparent;
+    border: solid $color-grey-4;
     border-width: 1px 0;
 
-    &:hover {
-        background-color: $color-input-focus;
-        border-color: darken($color-input-focus, 10%);
-
-        .sequence-member-inner {
-            > .struct-block > label,
-            > .char_field > label,
-            .sequence > label {
-                opacity: 1;
-            }
-        }
+    &:first-child {
+        border-top: 0;
     }
 
-    &:hover .object-help {
+    // Note this inherits .oject-help from page-editor.scss
+    .object-help {
```

Comment:
I think the more accurate answer is "not easily". We could change the classes to avoid any inheritance, which I could see would have benefits, but it could also add additional confusion to name-spacing/ ensure design consistency. It feels like a piece of work for once this PR is in when we can start thinking about improving the UI beyond StreamField.